### ISSUE TITLE
Error Loading on 0.4.2

### ISSUE THREAD

**Issue by geekofweek** [ollo69/ha-smartthinq-sensors#36]
Seem to have a problem running on 0.4.2, 0.4.1 works flawlessly.  Looks to error out attempting to load the devices.  Worth noting that both my Refrigerator and Dishwasher are only Tap-On devices, so while they are attached to my LG account they don't actually report back data over Wi-fi, they still get loaded though.

```
2020-05-05 12:49:56 INFO (MainThread) [custom_components.smartthinq_sensors] SmartThinQ client connected.
2020-05-05 12:49:56 INFO (MainThread) [custom_components.smartthinq_sensors] Starting LGE ThinQ devices...
2020-05-05 12:49:56 INFO (SyncWorker_38) [custom_components.smartthinq_sensors.wideq.core_v2] Loading model info for REFRIGERATOR. Model: Normal-101, Url: https://aic.lgthinq.com:46030/api/webContents/modelJSON?modelName=Normal-101&countryCode=WW&contentsId=JS0314074028491784&authKey=thinq
2020-05-05 12:49:57 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry LGE Devices for smartthinq_sensors
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 216, in async_setup
    hass, self
  File "/config/custom_components/smartthinq_sensors/__init__.py", line 163, in async_setup_entry
    lge_devices = await lge_devices_setup(hass, client)
  File "/config/custom_components/smartthinq_sensors/__init__.py", line 398, in lge_devices_setup
    await hass.async_add_executor_job(dev.init_device)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/smartthinq_sensors/__init__.py", line 280, in init_device
    self._model = f"{self._model}-{self._device.model_info.model_type}"
AttributeError: 'NoneType' object has no attribute 'model_type'
```

**Comment by ollo69** [comment]
Yes, I need to skip unsupported devices. 
Will fix in next release.

**Comment by ollo69** [comment]
Should be fixed with 0.4.3.

**Comment by geekofweek** [comment]
Looks good.  Props for the continued outstanding work on this.
